Cannot build Godot on Windows

**Operating system or device:**
Windows 10

**Issue description** (what happened, and what was expected):
Tried building the engine with Visual Studio 2015.

Got the following error:

Program Files (x86)\Windows Kits\10\include\10.0.10586.0\um" "/IC:\Program Files (x86)\Windows Kits\10\include\10.0.10586.0\winrt" "/IC:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE" "/IC:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\ATLMFC\INCLUDE" "/IC:\Program Files (x86)\Windows Kits\10\include\10.0.10586.0\ucrt" "/IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" "/IC:\Program Files (x86)\Windows Kits\10\include\10.0.10586.0\shared" "/IC:\Program Files (x86)\Windows Kits\10\include\10.0.10586.0\um" "/IC:\Program Files (x86)\Windows Kits\10\include\10.0.10586.0\winrt" /I "/IC:/Program Files/Microsoft DirectX SDK (March 2009)/Include" -DOC_X86_ASM -DDEBUG_MEMORY_ALLOC -DSCI_NAMESPACE /DMSVC /GR -DMUSEPACK_ENABLED -DOPENSSL_ENABLED -DSQUISH_ENABLED -DVORBIS_ENABLED -DOPUS_ENABLED -DTHEORA_ENABLED -DTHEORALIB_ENABLED -DPNG_ENABLED -DDDS_ENABLED -DPVR_ENABLED -DJPG_ENABLED -DWEBP_ENABLED -DSPEEX_ENABLED -DTOOLS_ENABLED -DGDSCRIPT_ENABLED -DMINIZIP_ENABLED -DXML_ENABLED -DETC1_ENABLED -DPNG_ARM_NEON_OPT=0 -DOPENSSL_NO_ASM -DOPENSSL_THREADS -DL_ENDIAN /Icore /Icore\math /Itools /Idrivers /I. /Iplatform\windows /Iplatform\windows\include /Itools\freetype /Itools\freetype\freetype\include /Idrivers\builtin_openssl2 /Idrivers\builtin_zlib\zlib /Idrivers\vorbis /Idrivers\builtin_openssl2\crypto /Idrivers\builtin_openssl2\openssl /Idrivers\builtin_openssl2\crypto\evp /Idrivers\builtin_openssl2\crypto\asn1 /Idrivers\builtin_openssl2\crypto\modes
cl @c:\users\jychong\appdata\local\temp\tmpdje4g5.lnk
o_str.c
drivers\builtin_openssl2\crypto\o_str.c(67): fatal error C1083: Cannot open include file: 'strings.h': No such file or directory
scons: *** [drivers\builtin_openssl2\crypto\o_str.windows.tools.obj] Error 2
scons: building terminated because of errors.

Any ideas why this happens? Or am I possibly building it incorrectly. I am using the standard instructions listed on your page for compilation under windows.

--- a/drivers/builtin_openssl2/crypto/o_str.c
+++ b/drivers/builtin_openssl2/crypto/o_str.c
@@ -64,7 +64,11 @@
 #if !defined(OPENSSL_IMPLEMENTS_strncasecmp) && \
     !defined(OPENSSL_SYSNAME_WIN32) && \
     !defined(NETWARE_CLIB)
-# include <strings.h>
+#ifdef _WIN32
+#include <string.h>
+#else
+#include <strings.h>
+#endif
 #endif
 
 int OPENSSL_strncasecmp(const char *str1, const char *str2, size_t n)

